client/src/components/Navbar.js

```diff
@@ -40,160 +44,135 @@ const useStyles = (theme) => ({
 
 function Navbar(props) {
   const [redirect, setRedirect] = useState(null);
+  const [loggedIn, setLoggedIn] = useState(false);
+  const [profileImg, setProfileImg] = useState(null);
+  const [anchorEl, setAnchorEl] = React.useState(null);
```

Comment:
Was this copy/pasted from somewhere? You have already defined useState we don't need the `React.` 😛

Reply:
Thats true.. and yes thats litarally from material-ui lib..lol sorry, i'll fix that